ResearchKit/Common/ORKChoiceAnswerFormatHelper.h

```diff
@@ -41,6 +42,7 @@ NS_ASSUME_NONNULL_BEGIN
 
 - (NSUInteger)choiceCount;
 
+- (id<ORKAnswerOption>)answerOptionAtIndex:(NSUInteger)index;
 - (nullable ORKImageChoice *)imageChoiceAtIndex:(NSUInteger)index;
 - (nullable ORKTextChoice *)textChoiceAtIndex:(NSUInteger)index;
```

Comment:
Do we need these two functions: `imageChoiceAtIndex:` and `textChoiceAtIndex:`, since now we have `answerOptionAtIndex:`? And if we choose to keep it, then we should also add convenient methods for all the other `validClasses`.